composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/SnackBarMessage.kt

```diff
@@ -0,0 +1,9 @@
+package org.ooni.probe.data.models
+
+sealed interface SnackBarMessage {
```

Comment:
This is not any SnackbarMessage, but the AddDescriptor specific messages. Maybe this could have a more specific name, or be inside the AddDescriptorViewModel class?